Kitodo/src/main/java/de/sub/goobi/helper/Helper.java

```diff
@@ -464,9 +464,27 @@ public void update(Observable o, Object arg) {
         }
     }
 
+    /**
+     * Get current logged in user.
+     * 
+     * @return current logged in user
+     */
     public static User getCurrentUser() {
-        LoginForm login = (LoginForm) Helper.getManagedBeanValue("#{LoginForm}");
-        return login != null ? login.getMyBenutzer() : null;
+        if (Objects.isNull(currentUser)) {
+            LoginForm login = (LoginForm) Helper.getManagedBeanValue("#{LoginForm}");
+            currentUser = login != null ? login.getMyBenutzer() : null;
+        }
+        return currentUser;
+    }
+
+    //TODO: find way to test without this method - faces
```

Comment:
Is this TODO correct? It looks a little bit wrong or need a better description.

Reply:
It is correct, it would be better to get this information from LoginForm (faces context ) instead set up this value explicitly for tests.